python/lsst/ts/m2com/utility.py

```diff
@@ -123,6 +125,41 @@ def read_yaml_file(filepath: str | Path) -> dict:
     return content
 
 
+def read_error_code_file(filepath: str | Path) -> dict:
+    """Read the error code file.
+
+    Parameters
+    ----------
+    filepath : `str` or `pathlib.PosixPath`
+        TSV file path.
+
+    Returns
+    -------
+    content : `dict`
+        File content.
+
+    Raises
+    ------
+    `IOError`
+        Cannot open the file.
+    """
+
+    content = dict()
+    try:
+        with open(filepath, "r") as file:
+            csv_reader = csv.reader(file, delimiter="\t")
+
+            # Get the error details
+            for row in csv_reader:
+                if row[0].isdigit():
+                    content[row[0]] = row[1:]
+
+    except IOError:
```

Comment:
Is there any particular reason for capturing and rephrasing the same error? Is the message from the original exception not clear enough that you rather rephrase it?

Also, if you really want to re-raise the exception. Consider using a `RuntimeError` instead..

Reply:
Fixed in the commit: [9251138](https://github.com/lsst-ts/ts_m2com/pull/29/commits/925113864e2c5d488004305866687b3522d84d20). Thanks!